Sources/OpenCombineFoundation/Timer+Publisher.swift

```diff
@@ -337,11 +337,19 @@ extension RunLoop.Mode {
         return CFRunLoopMode(rawValue as CFString)
 #else
         return rawValue.withCString {
-            CFStringCreateWithCString(
+#if swift(>=5.3)
+            return CFStringCreateWithCString(
+                nil,
+                $0,
+                CFStringBuiltInEncodings.UTF8.rawValue
+            )
+#else
+            return CFStringCreateWithCString(
                 nil,
                 $0,
                 CFStringEncoding(kCFStringEncodingUTF8)
             )
+#endif
```

Comment:
Let's extract the encoding into a local variable so there's less code duplication:

```swift
#if swift(>=5.3)
let encoding = CFStringBuiltInEncodings.UTF8.rawValue
#else
let encoding = CFStringEncoding(kCFStringEncodingUTF8)
#endif // swift(>=5.3)

return CFStringCreateWithCString(
    nil,
    $0,
    encoding
)
```